packages/driver/cypress/e2e/cypress/stack_utils-invocationDetails.cy.ts

```diff
@@ -0,0 +1,98 @@
+import type { InvocationDetails } from '../../../src/cypress/stack_utils'
+
+// Note: the tests in this spec assert against their own invocation details. So if any of the line numbers change in this file, the assertions will need to be updated.
+describe('stack_utils getInvocationDetails', () => {
+  context('basic test invocation', () => {
+    it('correctly extracts invocation details for Chrome', { browser: 'chrome' }, function () {
```

Comment:
In CI we test against multiple different chromium browsers, so we don't want to limit this to just `chrome`. Most importantly, we test against Chrome Beta so that would let us know early if the stack changed. Honestly, it may just be easier to flex the assertions in the test itself.